docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -20,272 +20,136 @@ Current binder implementations include:
-|partition-key-expression
-|null
-|A SpEL expression that determines how to partition outbound data.
-When interface `PartitionKeyExtractorStrategy` is not implemented, it will be called in the method `PartitionHandler#extractKey`.
+|Properties | Type |Description
+
+|*spring.cloud.azure.eventhubs*.connection-string | String | Event Hubs Namespace connection string value.
+|*spring.cloud.azure.eventhubs*.namespace | String | Event Hubs Namespace value.
+|*spring.cloud.azure.eventhubs*.domainName | String | Domain name of an Azure Event Hubs Namespace value.
+|*spring.cloud.azure.eventhubs*.eventHubName | String | Name of an Event Hub entity.
+|*spring.cloud.azure.eventhubs*.customEndpointAddress | String | Custom Endpoint address.
+|*spring.cloud.azure.eventhubs*.isSharedConnection | Boolean | Whether to use the same connection for different Event Hub producer / consumer client.
+|*spring.cloud.azure.eventhubs*.processor.checkpointStore.* | NA | Blob checkpoint store configuration options.
+|*spring.cloud.azure.eventhubs*.processor.checkpointStore.createContainerIfNotExists | Boolean | If allowed to create container if not exists.
+|*spring.cloud.azure.eventhubs*.processor.checkpointStore.customerProvidedKey | String | Base64 encoded string of the encryption key.
+|*spring.cloud.azure.eventhubs*.processor.checkpointStore.encryptionScope | String | Encryption scope to encrypt blob contents on the server.
+|*spring.cloud.azure.eventhubs*.processor.checkpointStore.serviceVersion | BlobServiceVersion | The versions of Azure Storage Blob supported by this client library.
+|*spring.cloud.azure.eventhubs*.processor.checkpointStore.blobName | String | Storage blob name.
+|*spring.cloud.azure.eventhubs*.processor.checkpointStore.containerName | String | Storage container name.
 |===
 
-For more information about setting partition for the producer properties, please refer to the https://docs.spring.io/spring-cloud-stream/docs/current/reference/html/spring-cloud-stream.html#_producer_properties[Producer Properties of Spring Cloud Stream].
-
-===== Event Hub Producer Properties
+===== Azure Event Hubs Binding Configuration Options
+Below options are used to configure Azure Event Hubs Binding properties.
 
-It supports the following configurations with the format of `spring.cloud.stream.eventhubs.bindings.&lt;channelName&gt;.producer`.
-[cols="3*", options="header"]
 |===
-|PropertyName |Default Value | Description
-|sync
-|false
-|Whether the producer should act in a synchronous manner with respect to writing messages into a stream. If true, the
-producer will wait for a response from Event Hub after a send operation.
-|send-timeout
-|10000
-|Effective only if `sync` is set to true. The amount of time to wait for a response from Event Hub after a send operation, in milliseconds.
+|Properties | Type |Description
+|*spring.cloud.stream.eventhubs*.binding | EventHubsExtendedBindingProperties | eventhubs binding additional properties.
```

Comment:
Please list all consumer and producer properties here, you can refer to classes of EventHubsConsumerProperties and EventHubsProducerProperties

Reply:
updated